cli/cmd/turbo/main.go

```diff
@@ -57,15 +60,25 @@ func main() {
 	c.HelpWriter = os.Stdout
 	c.ErrorWriter = os.Stderr
 	// Parse and validate cmd line flags and env vars
+	// Note that cf can be nil
 	cf, err := config.ParseAndValidate(c.Args, ui, turboVersion)
 	if err != nil {
 		ui.Error(fmt.Sprintf("%s %s", uiPkg.ERROR_PREFIX, color.RedString(err.Error())))
 		os.Exit(1)
 	}
+
+	var logger hclog.Logger
```

Comment:
should we move this into config.ParseAndValidate?

Reply:
`config.ParseAndValidate` doesn't always return a config, even if there is no error. It bails early on things like `--help` and `--version`. 

A more proper refactor would probably change that, and then only instantiate `process.Manager` for commands where we also need a `Config` instance. 

As a temporary measure, we could also have `config.ParseAndValidate` return a `hclog.Logger` instance, as long as there is no error. I'm not sure that's a lot better than the current situation, but it does at least move it to where the decision on returning config is.

What do you think?

Reply:
Got it. Let's table for now then.